projects/qgrid-core/body/body.let.js

```diff
@@ -17,6 +18,23 @@ export class BodyLet {
 				}
 			});
 
+		observe(model.mouseChanged)
+			.subscribe(({ state }) => {
+				const { code, status, target } = state;
+				if (target && code === 'right' && status === 'up') {
+					this.targetElement = target.element;
+					element.classList.add('q-grid-can-select-text');
```

Comment:
from where you get element? i don't see it

Reply:
oops. Sorry, just a typo